crates/bevy_text/src/font_atlas.rs

```diff
@@ -1,47 +1,23 @@
-use ab_glyph::{GlyphId, Point};
 use bevy_asset::{Assets, Handle};
-use bevy_math::Vec2;
+use bevy_math::{IVec2, Vec2};
 use bevy_render::{
     render_resource::{Extent3d, TextureDimension, TextureFormat},
     texture::Image,
 };
 use bevy_sprite::{DynamicTextureAtlasBuilder, TextureAtlas};
 use bevy_utils::HashMap;
 
-#[cfg(feature = "subpixel_glyph_atlas")]
-#[derive(Copy, Clone, Eq, PartialEq, Hash, Debug)]
-pub struct SubpixelOffset {
-    x: u16,
-    y: u16,
-}
-
-#[cfg(feature = "subpixel_glyph_atlas")]
-impl From<Point> for SubpixelOffset {
-    fn from(p: Point) -> Self {
-        fn f(v: f32) -> u16 {
-            ((v % 1.) * (u16::MAX as f32)) as u16
-        }
-        Self {
-            x: f(p.x),
-            y: f(p.y),
-        }
-    }
-}
-
-#[cfg(not(feature = "subpixel_glyph_atlas"))]
-#[derive(Copy, Clone, Eq, PartialEq, Hash, Debug)]
-pub struct SubpixelOffset;
-
-#[cfg(not(feature = "subpixel_glyph_atlas"))]
-impl From<Point> for SubpixelOffset {
-    fn from(_: Point) -> Self {
-        Self
-    }
-}
+use crate::GlyphAtlasLocation;
 
+/// Rasterized glyphs are cached, stored in, and retrieved from, a `FontAtlas`.
+///
+/// A [`FontAtlasSet`](crate::FontAtlasSet) contains one or more `FontAtlas`es.
 pub struct FontAtlas {
```

Comment:
Debug derive at least please.

Reply:
manually implemented debug to skip the dynamictextureatlasbuilder